docs/FAQ.md

```diff
@@ -96,8 +96,8 @@ will use a bottled version of $FORMULA, but
 `brew install $FORMULA --enable-bar` will trigger a source build.
 * The `--build-from-source` option is invoked.
 * The environment variable `HOMEBREW_BUILD_FROM_SOURCE` is set.
-* The machine is not running OS X 10.9+ as all bottled builds are
-generated on Mavericks or later.
+* The machine is not running OS X 10.10+ as all bottled builds are
```

Comment:
`The machine is not running a supported version of macOS as all bottled builds are generated only for supported macOS versions.`